src/jquery.ime.inputmethods.js

```diff
@@ -198,6 +198,10 @@
 			name: 'फोनेटिक',
 			source: 'rules/hi/hi-phonetic.js'
 		},
+		'hoc': {
+			name: '𑢹𑣉𑣉/Ho',
+			source: 'rules/hoc/hoc-transliteration.js'
```

Comment:
Did you try to test the input method? If not please do. This file rules/hoc/hoc-transliteration.js does not exist. The file you wrote the rules are just rules/hoc.